Gemfile

```diff
@@ -175,10 +175,14 @@ group :development do
 
   # Validate missing i18n keys
   gem 'i18n-tasks', '~> 1.0', require: false
+end
 
+group :development, :test do
   # Profiling tools
   gem 'memory_profiler', require: false
+  gem 'ruby-prof', require: false
   gem 'stackprof', require: false
+  gem 'test-prof'
```

Comment:
Should this one be `require: false` too? Wasn't sure if it was another one of the CLI tools like the other

Reply:
No - it's correct as-is. There are a few ways to use its various features, but some of them are env var activated and it needs to be required to see that and do its thing.

Reply:
OK, I thought it might have been one of those more like https://github.com/mastodon/mastodon/blob/25c66fa640962a4d54d59a3f53516ab6dcb1dae6/spec/spec_helper.rb#L5-L14 that just loads itself if one of those flags is set